.github/workflows/push-docker-local-image.yaml

```diff
@@ -0,0 +1,76 @@
+name: publish docker image
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - alpha
+  release:
+    types: [published]
+  workflow_call:
+    secrets:
+      GH_DOCKER_ACCESS_USER:
+        required: true
+      GH_DOCKER_ACCESS_TOKEN:
+        required: true
+
+env:
+  DOCKER_REPOSITORY: "tigrisdata/tigris"
```

Comment:
We probably want to push to a different repo tigrisdata/tigris-allinone or tigris-local or something like that.